src/main/java/org/mule/extensions/jms/internal/common/JmsCommons.java

```diff
@@ -108,26 +110,13 @@ public static void evaluateMessageAck(InternalAckMode ackMode, JmsSession sessio
    * @throws JMSException If an error happens creating a new {@link JmsSession}
    */
   public static JmsSession createJmsSession(JmsConnection jmsConnection, InternalAckMode ackMode, boolean isTopic,
-                                            JmsSessionManager jmsSessionManager)
+                                            JmsSessionManager jmsSessionManager, OperationTransactionalAction transactionalAction)
```

Comment:
don't you then have to add OperationTransactionalAction to the list of non advertised parameters?

Reply:
is advertised, the parameter must appear in the ext model and the XSD.
There is logic in the SDK, that was done a long time ago, that contributes it with a description, optionality, fixes the name, etc, etc.